app/views/claims/_personal_details.html.erb

```diff
@@ -11,7 +11,7 @@
     <div class="govuk-summary-list__row">
       <dt class="govuk-summary-list__key">Date of birth</dt>
       <dd class="govuk-summary-list__value">
-        <%= l(Date.parse(@teacher_id_user_info["birthdate"])) %>
+        <%= l(Date.parse(@teacher_id_user_info["birthdate"])) if @teacher_id_user_info["birthdate"] %>
       </dd>
     </div>
```

Comment:
I think we would want to hide the whole row in this case, rather than showing a row with a blank value.

Reply:
Good point. Done! Thanks